api/download.py

```diff
@@ -266,6 +265,9 @@ def _find_new_path(path, ids_of_paths, _id):
             path = container['uid']
         if not path and container.get('code'):
             path = container['code']
+
+        path = path.encode('ascii', 'ignore')
```

Comment:
I did the same in reaper lately, I think `errors='ignore'` is a bit easier on the eyes.